### PR TITLE
Separate Production/Staging Github workflow

### PR DESCRIPTION
**Pull request by oren-akto**
Seperated the existing Github workflow into two files:

- staging.yml - push docker images tagged as akto1, akto2, akto3
- prod.yml - push docker image tagged as local

### CODE REVIEW

File: .github/workflows/staging.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
+name: Staging
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
```
Comment (ankush-jain-akto) [inline .github/workflows/staging.yml:10]: Switch this off on pull-request and push? This should be triggered only when triggered manually

File: .github/workflows/prod.yml [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+name: Production
 
 # Controls when the workflow will run
 on:
   # Triggers the workflow on push or pull request events but only for the master branch
   push:
     branches: [ master ]
   pull_request:
-    branches: [ master, develop ]
+    branches: [ master ]
```
Comment (ankush-jain-akto) [inline .github/workflows/prod.yml:9]: Switch this off on pull-request and push? This should be triggered only when triggered manually

File: .github/workflows/staging.yml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      Tag:
+        required: true
+        type: choice
+        options:
+          - akto1
```
Comment (ankush-jain-akto) [inline .github/workflows/staging.yml:18]: Rename this to akto_staging1 and so on please ?

File: .github/workflows/prod.yml (hunk 2 above)
Comment (oren-akto) [reply .github/workflows/prod.yml:9]: Should I remove push/pull triggers for the staging.yml as well?

File: .github/workflows/staging.yml [hunk 4]
```diff
@@ -0,0 +1,93 @@
+name: Staging
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ develop ]
```
Comment (ankush-jain-akto) [inline .github/workflows/staging.yml:7]: leave master here. Any push to master should run the CI. 
This one doesn't do just "staging". This CI compiles the code on every pull-request and push to develop & master. Keep master in both - push and develop

**Review by ankush-jain-akto — APPROVED** [review]

**Pull request merged**